sw/device/silicon_creator/mask_rom/mock_romextimage_ptrs.h

```diff
@@ -0,0 +1,43 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_MASK_ROM_MOCK_ROMEXTIMAGE_PTRS_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_MASK_ROM_MOCK_ROMEXTIMAGE_PTRS_H_
+
+#include "sw/device/lib/testing/mask_rom_test.h"
+#include "sw/device/silicon_creator/mask_rom/romextimage_ptrs.h"
+
+namespace mask_rom_test {
+namespace internal {
+
+/**
+ * Mock class for romextimage_ptrs.h
+ */
+class MockRomextimagePtrs {
+ public:
+  MOCK_METHOD(const manifest_t *, slot_a_manifest_ptr_get, ());
```

Comment:
I am not excited about using a mock here, since this is going to be used as a fake rather than a mock, but I guess it's the price we pay for not having amazing infrastructure...